keyboards/keyprez/unicorn/unicorn.c

```diff
@@ -0,0 +1,4 @@
+// Copyright 2022 Christian Sandven (@csandven)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "unicorn.h"
```

Comment:
This allows for a default behavior to be defined, without the need for the user to specify the behavior.  This is very useful for the configurator which doesn't have a way to configure this behavior, currently.

```suggestion
#include "unicorn.h"

#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) {
        if (clockwise) {
            tap_code_delay(KC_VOLU, 10);
        } else {
            tap_code_delay(KC_VOLD, 10);
        }
    }
    return true;
}
#endif
```